resources/public/api/conf/common/testing.md

```diff
@@ -1,11 +1,3 @@
-You can use the sandbox environment to <a href="/api-documentation/docs/testing">test this API</a>. You can use the <a href="/api-documentation/docs/api/service/api-platform-test-user/1.0">Create Test User API</a> or it's frontend service to create test users.
+You can use the Sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing). You can test different scenarios in the Sandbox by passing the Gov-Test-Scenario header. The Sandbox also allows you to perform [stateful and dynamic testing](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/documentation/how-to-integrate.html#sandbox-testing) for some APIs. The ‘Test data’ section under each endpoint explains the test scenarios that can be simulated.
```

Comment:
Environment specific url can be used: `[test this API](/api-documentation/docs/testing)`